[2024-04-24T16:49:15.851+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-24T16:49:16.042+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_orders manual__2024-04-24T16:48:50.005589+00:00 [queued]>
[2024-04-24T16:49:16.200+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_orders manual__2024-04-24T16:48:50.005589+00:00 [queued]>
[2024-04-24T16:49:16.211+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-24T16:49:16.532+0000] {taskinstance.py:2327} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_file_orders> on 2024-04-24 16:48:50.005589+00:00
[2024-04-24T16:49:16.739+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_orders', 'manual__2024-04-24T16:48:50.005589+00:00', '--job-id', '288', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmppxt9282b']
[2024-04-24T16:49:16.759+0000] {standard_task_runner.py:91} INFO - Job 288: Subtask upload_file_orders
[2024-04-24T16:49:16.785+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4175) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-24T16:49:16.786+0000] {standard_task_runner.py:63} INFO - Started process 4283 to run task
[2024-04-24T16:49:17.209+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_orders manual__2024-04-24T16:48:50.005589+00:00 [running]> on host 204d0a7369e0
[2024-04-24T16:49:20.238+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_orders' AIRFLOW_CTX_EXECUTION_DATE='2024-04-24T16:48:50.005589+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-24T16:48:50.005589+00:00'
[2024-04-24T16:49:20.239+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-24T16:49:20.279+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-24T16:49:20.354+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-04-24T16:49:23.027+0000] {gcs.py:557} INFO - File /opt/***/dags/data/olist_orders_dataset.csv uploaded to olist_orders_dataset.csv in swu-ds-525-112233 bucket
[2024-04-24T16:49:23.029+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-24T16:49:23.060+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_orders, execution_date=20240424T164850, start_date=20240424T164916, end_date=20240424T164923
[2024-04-24T16:49:23.196+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-24T16:49:27.457+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-24T16:49:27.480+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
